scripts/eurostat/gdp/README.md

```diff
@@ -0,0 +1,56 @@
+# Importing Eurostat yearly GDP into Data Commons
+
+Author: fpernice-google
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Procedure](#Import-Procedure)
+
+## About the Dataset
+
+### Download URL
+
+ZIP file is available for download from https://ec.europa.eu/eurostat/data/database.
+
+### Overview
+
+> Gross Domestic Product (GDP) measures the overall level of economic activity in a country or region. The Eurostat publishes various forms of economic data, including GDP, inflation, and employment. Specifically on GDP, it publishes data at various levels of government, and with different measurement styles. We import all Eurostat GDP data.
```

Comment:
Is the initial ">" here and below from GitHub or in the markdown?  What is its purpose?